app/assets/javascripts/parsers.js

```diff
@@ -96,8 +96,40 @@ $(function() {
     }
   });
 
+
+
   return $('#parsers').dataTable({
-    "aaSorting": [[4, 'desc']]
+    "processing": true,
+    "serverSide": true,
+    "ajax": "/parsers/datatable",
+    "aaSorting": [[4, 'desc']],
+    "columns": [
+      {
+        "data": "name",
+        "render": function(data, type, row, meta) {
+          return `<a href="/parsers/${row.id}/edit">${data}</a>`;
+        }
+      },
+      { "data": "strategy" },
+      {
+        "data": "partner_name",
+        "render": function(data, type, row, meta) {
+          debugger;
```

Comment:
perhaps remove the debugger?

Reply:
Yeah I missed this one during my `git add -p` apparently. Removed :)